util/image_viewer/slip_viewer/rotate_detector.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import division, print_function
+import sys
+from dxtbx.model.experiment_list import ExperimentListFactory
+from scitbx.matrix import col
+
+"""
```

Comment:
For info - what is the purpose of this example script?

Reply:
Right, this is just for testing.  You run this script to rotate a detector model then load the resultant experiment file in the image viewer.

I have another issue I'll submit soon that will use this script too.  After these are finished I'm cool with this example script going away or landing on a tutorial page or something.